geoengine/workflow.py

```diff
@@ -201,7 +201,7 @@ def wms_get_map_as_image(self, bbox: QueryRectangle, colorizer_min_max: Tuple[fl
 
     def __wms_get_map_request(self,
                               bbox: QueryRectangle,
-                              colorizer_min_max: Tuple[float, float] = None) -> req.PreparedRequest:
+                              colorizer) -> req.PreparedRequest:
```

Comment:
Please take a Colorizer instance, e.g.

```suggestion
                              colorizer: Colorizer) -> req.PreparedRequest:
```

and then just call `to_query_string` within this method